onward/app/controllers/MediaInSectionController.scala

```diff
@@ -40,14 +41,16 @@ class MediaInSectionController(
   private def lookup(edition: Edition, mediaType: String, sectionId: String, seriesId: Option[String])(implicit
       request: RequestHeader,
   ): Future[Option[Seq[RelatedContentItem]]] = {
-    val currentShortUrl = request.getQueryString("shortUrl").getOrElse("")
+    val currentShortUrl = request.getQueryString("shortUrl")
     log.info(s"Fetching $mediaType content in section: $sectionId")
 
     val excludeTags: Seq[String] = (request.queryString.getOrElse("exclude-tag", Nil) ++ seriesId).map(t => s"-$t")
     val tags = (s"type/$mediaType" +: excludeTags).mkString(",")
 
     def isCurrentStory(content: ApiContent) =
-      content.fields.flatMap(_.shortUrl).exists(!_.equals(currentShortUrl))
```

Comment:
This method was incorrectly named  - it's actually `isNotCurrentStory` - but I've switched the logic around rather than change the name for consistency.